kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -807,26 +196,32 @@ window.ExerciseView = Backbone.View.extend({
         var self = this;
 
         var defaults = {
-            seed: Math.floor(Math.random() * 200),
-            framework: "khan-exercises"
+            seed: Math.floor(Math.random() * 200)
         };
 
-        var question_data = $.extend(defaults, question_data);
+        question_data = $.extend(defaults, question_data);
 
         this.data_model.set(question_data);
 
         this.$("#workarea").html("<center>" + gettext("Loading...") + "</center>");
 
+
+        // Khan Exercises now moves the solution area around and fails to put it back again after
```

Comment:
Put the solution area back in place after Khan Exercises picks it up, shakes it about, and then fails to put it back again.